Firestore/Swift/Tests/API/BasicCompileTests.swift

```diff
@@ -63,7 +63,6 @@ func initializeDb() -> Firestore {
   let settings = FirestoreSettings()
   settings.host = "localhost"
   settings.isPersistenceEnabled = true
-  settings.areTimestampsInSnapshotsEnabled = true
```

Comment:
There's apparently no way to suppress deprecation warnings in swift, so I just removed the setting.